tools/lsp/preview/connector/wasm.rs

```diff
@@ -93,6 +99,16 @@ impl PreviewConnector {
                     } else {
                         match ui::create_ui(&to_lsp, &style, experimental) {
                             Ok(ui) => {
+                                let api = ui.global::<crate::preview::ui::Api>();
+
+                                api.set_runs_in_slintpad(true);
```

Comment:
How do you distinguish between SlintPad and the wasm preview embedded in vscode?